lib/typing_env.ml

```diff
@@ -117,12 +125,15 @@ let pair_items ~reference ~current =
       | _ -> subst)
     Subst.identity current
 
-let for_diff ~reference ~current =
+let initialized_env () =
   Compmisc.init_path ();
```

Comment:
We should be a bit careful with this, I think it should only be called once. Might be harmless to call it several times but to err on the safe side I'd make sure we only do it once!

Reply:
I did it with closures, I was wondering if it would be any better to do it with lazy values ?
`let initialized_env = lazy (Compmisc.init_path (); Compmisc.initial_env ())`

Reply:
Both are fine by me!